pontoon/teams/views.py

```diff
@@ -306,15 +322,15 @@ def ajax_translation_memory(request, locale):
         {
             "locale": locale,
             "search_query": search_query,
-            "tm_entries": page,
-            "has_next": page.has_next(),
+            "tm_entries": list(combined_entries),
```

Comment:
This is already a list.
```suggestion
            "tm_entries": combined_entries,
```